src/main.rs

```diff
@@ -22,6 +22,7 @@ FLAGS:
     -f, --full-screen    Show scout in full screen (default)
     -h, --help           Prints help information
     -i, --inline         Show scout under the current line
+    -n, --no-sort        Do not sort the result by score
```

Comment:
I think this is a very good addition, but I'm not so sure about the flag name. I'm thinking we can use:

* `-S, --skip-sort`, but this can be confusing since there is already a `-s` option as you pointed out
* `-u, --unsort` or `-u, --unrank`
* `-p, --preserve`, as in "preserve order" or "keep order". I guess `-k` could work in that regard too

I'm open to other suggestions, tho

Reply:
hm, `-p` is probably my favorite of these. I don't really have strong feelings on the matter though.

Reply:
If you tell me which one to use, I'd be happy to change PR :)

Reply:
@mullr I think `-p, --preserve-order` is a good option, let's go with that one

Reply:
Done!

Reply:
@mullr thanks for the changes! I'm currently with very limited internet access so it's going to take a while until I manage to merge and release the change, but I'll do it eventually.